clearblade.md

```diff
@@ -2760,7 +2760,8 @@ This method removes all the permissions for a resource.
 
 Class: ClearBlade.Lock()
 
-This class allows for interacting with cache locks.
+This class allows for interacting with cache locks. The lock name must be included where the name is asked for in the code service. 
```

Comment:
This class allows for locking ClearBlade functions (only sync). If the lock is being used across multiple services, they must all use the same lock name